.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report
+about: Report a bug encountered while operating Kubernetes federation v2
```

Comment:
/Kubernetes federation/Federation/

I think 'Federation v2' is the name of the project rather than 'Kubernetes Federation v2', at least for now, and that this should be reflected in all the templates in this PR.